trestle/core/control_context.py

```diff
@@ -53,6 +53,7 @@ class ControlContext:
     allowed_sections: Optional[str] = None
     comp_def: Optional[comp.ComponentDefinition] = None
     comp_name: Optional[str] = None
+    inherited_props: Optional[Dict[str, Any]] = None
```

Comment:
What is Any here?

Reply:
The Any is itself a recursive Dict of [str, Dict[str, Dict[str]]]...] - so I think this is ok.